.kiro/specs/get-streams-by-sender/design.md

```
@@ -0,0 +1,181 @@
+async function fetchSenderStreams(sender: string): Promise<bigint[]> {
+  // Returns an array of stream IDs (u64) for the given sender.
+  // Returns [] if the sender has no streams — never throws.
+  const streamIds = await streamContract.getStreamsBySender({ sender });
+  return streamIds; // bigint[]
+}
+
+// Example: load and display all streams for a wallet address
+const ids = await fetchSenderStreams("GABC...XYZ");
+console.log(`Sender has ${ids.length} streams: ${ids.join(", ")}`);
+```
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)